client/src/components/Public/EntitySearchAutocomplete/EntitySearchAutocomplete.js

```diff
@@ -160,22 +166,21 @@ export default compose(
     },
     getItemValue: () => (entity) => (entity.name ? entity.name : ''),
     renderItem: () => (entity, isHighlighted) => (
-      <div key={entity.eid} className={entity.name && classnames('item', isHighlighted && 'item--active')} >
+      <div
+        key={entity.eid}
+        className={
+          entity.name &&
+          classnames('autocomplete-item', isHighlighted && 'autocomplete-item--active')
```

Comment:
This is the worst use of `classnames` I've ever seen :smile: (I know it's not your code...)